jibri-xmpp-client/app.py

```diff
@@ -315,6 +329,18 @@ def jibri_start_callback(client, url, stream_id, sipaddress=None, displayname=No
     stream_id=stream_id.strip()
     sipaddress=sipaddress.strip()
 
+    #assume we're streaming if a stream_id is provided
+    if stream_id:
+        recording_mode='stream'
+
+    #default to file-based recording if none is specified
+    if not recording_mode:
+        recording_mode = 'file'
+
+    #clear stream id if file mode is set
+    if recording_mode == 'file':
```

Comment:
did you consider treating this as an error?  (setting `stream_id` when `recording_mode` is `file`) wonder if it might be clearer